src/components/AcceptAllOrMoreChoices.tsx

```diff
@@ -47,8 +47,13 @@ export function AcceptAllOrMoreChoices({
           </p>
         </div>
         <div>
-          <p className="paragraph">
-            {formatMessage(messages.acceptAllDescription)}
+        <p className="paragraph">
+            <div
+              // eslint-disable-next-line react/no-danger
+              dangerouslySetInnerHTML={{
+                __html: formatMessage(messages.acceptAllDescription),
```

Comment:
Generally it's a bad idea to dangerouslySetInnerHTML for user-generated content (hence the warning). I'm not sure if we can trust our customers here tbh, so if there's a way to do this without allowing raw HTML injection I'd prefer it. I'm not sure what that would be though

Reply:
Would that be something along the lines of only detecting precisely only urls and creating a href for them? If we're trusting href, that is.

Reply:
Just playing devils advocate, I wouldn't think of contradicting the security take here, but it looks like we gave the DoNotSellExplainer a pass for this. I'm not sure if this view is much more used, that might be the concern there, and I understand that we also might be wary of starting to single case every view... I'm just trying to keep Shutterfly happy, I'm going to ask them how they would feel about leaving the url in plain text.

Reply:
allowing user generated html without sanitizing it first opens the door to XSS attacks and other nefarious things. This is running on their site though, so I think it's probs less of an issue than in the AD. If they wanted to XSS themselves on their own site there's easier ways I guess. I'm not comfortable making the final OK call here but if we could get someone more familiar with this repo to look this over I'm ok with it

Reply:
@banzzai mind spinning up a ticket? Would be helpful to understand the customer request motivating this change (but in a private forum / not in this open source repo). Depending on the request, there may be a safer method available.

Reply:
Fortunately airgap can regulate the network requests made by this html. Given that this is customer provided and we have airgap as a backup, I'm fine with no sanitization